internal/logging/context.go

```diff
@@ -19,6 +24,9 @@ func init() {
 		panic(err)
 	}
 	globalLogger.Logger.SetLevel(level)
+	SetKLogLevel(os.GetEnvInt("KLOG_LEVEL", 0))
+
+	runtimelog.SetLogger(logrusr.New(globalLogger))
```

Comment:
`sigs.k8s.io/controller-runtime/pkg/log` wants us to set a logger of type `go-logr/logr`. We currently use `github.com/sirupsen/logrus`. 

`github.com/bombsimon/logrusr` is a small adapter that returns a `go-logr/logr` which is implemented by `github.com/sirupsen/logrus`.